x-pack/legacy/plugins/siem/public/containers/case/api.test.tsx

```diff
@@ -0,0 +1,402 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { KibanaServices } from '../../lib/kibana';
+import {
+  deleteCases,
+  getActionLicense,
+  getCase,
+  getCases,
+  getCasesStatus,
+  getCaseUserActions,
+  getReporters,
+  getTags,
+  patchCase,
+  patchCasesStatus,
+  patchComment,
+  postCase,
+  postComment,
+  pushCase,
+  pushToService,
+} from './api';
+import {
+  actionLicenses,
+  allCases,
+  basicCase,
+  allCasesSnake,
+  basicCaseSnake,
+  actionTypeExecutorResult,
+  pushedCaseSnake,
+  casesStatus,
+  casesSnake,
+  cases,
+  caseUserActions,
+  pushedCase,
+  pushSnake,
+  respReporters,
+  serviceConnector,
+  casePushParams,
+  tags,
+  caseUserActionsSnake,
+  casesStatusSnake,
+} from './mock';
+import { CASES_URL } from './constants';
+import { DEFAULT_FILTER_OPTIONS, DEFAULT_QUERY_PARAMS } from './use_get_cases';
+
+const abortCtrl = new AbortController();
+const mockKibanaServices = KibanaServices.get as jest.Mock;
+jest.mock('../../lib/kibana');
+
+const fetchMock = jest.fn();
+mockKibanaServices.mockReturnValue({ http: { fetch: fetchMock } });
+
+describe('Case Configuration API', () => {
+  describe('deleteCases', () => {
+    beforeEach(() => {
+      fetchMock.mockClear();
+      fetchMock.mockResolvedValue('true');
```

Comment:
If I remember correctly the API returns an empty response, right?

Reply:
it's weird, it doesnt return {} as i expected. it returns an empty string. i can change this to empty string. to see this, put a `console.log('response', typeof response);` on line 213 of `containers/case/api` and delete a case in the ui

Reply:
You are right. The server responds with empty content. Probably it's converted to an empty string.